src/h5web/toolbar/controls/DownloadBtn.tsx

```diff
@@ -0,0 +1,53 @@
+import type { AriaAttributes } from 'react';
+import type { IconType } from 'react-icons/lib';
+import styles from '../Toolbar.module.css';
+
+interface Props extends AriaAttributes {
+  label: string;
+  icon?: IconType;
+  iconOnly?: boolean;
+  // If specified, browser will prompt user to download the file instead of opening a new tab
+  filename?: string;
+  // Invoked on click; must return URL to set as `href`, or `false` to do nothing
+  getDownloadUrl: () => string | false;
+}
+
+function DownloadBtn(props: Props) {
```

Comment:
The CSV export button behaves the same way as the `SnapshotBtn`, so I thought I'd make an abstraction as well.